store/index.js

```diff
@@ -30,12 +34,14 @@ export const mutations = {
 
 export const getters = {
 
+    seasonInfo: state => state.seasonInfo,
+
     currentSeason: (state) => {
-        return state.currentSeason || '';
+        return state.seasonInfo.currentSeason || '';
```

Comment:
Codacy has a fix for the issue: [Replace `''` with `""`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
        return state.seasonInfo.currentSeason || "";
```